VideoApp/VideoApp/ViewControllers/Settings/BandwidthProfile/BandwidthProfileSettingsViewModel.swift

```diff
@@ -19,14 +19,27 @@ import Foundation
 class BandwidthProfileSettingsViewModel: SettingsViewModel {
     let title = "Bandwidth Profile"
     var sections: [SettingsViewModelSection] {
-        [
+        let editMaxSubscriptionBitrateViewModel = editMaxSubscriptionBitrateViewModelFactory.makeEditTextViewModel()
+        let editMaxTracksViewModel = editMaxTracksViewModelFactory.makeEditTextViewModel()
+        
+        return [
             .init(
                 rows: [
                     .optionList(
                         title: "Mode",
                         selectedOption: appSettingsStore.bandwidthProfileMode.title,
                         viewModelFactory: selectBandwidthProfileModeViewModelFactory
                     ),
+                    .editableText(
+                        title: editMaxSubscriptionBitrateViewModel.title,
+                        text: editMaxSubscriptionBitrateViewModel.text.nilIfEmpty ?? editMaxSubscriptionBitrateViewModel.placeholder,
+                        viewModelFactory: editMaxSubscriptionBitrateViewModelFactory
+                    ),
+                    .editableText(
+                        title: editMaxTracksViewModel.title,
+                        text: editMaxTracksViewModel.text.nilIfEmpty ?? editMaxTracksViewModel.placeholder,
+                        viewModelFactory: editMaxTracksViewModelFactory
```

Comment:
I did this a little differently than other settings. It is not perfect but it is a step in the direction of being able to use a single type to describe a setting which is used for both the screen that shows the setting in a cell and the screen to modify the setting.